README.md

```diff
@@ -81,6 +81,7 @@ Talks:
 
 - 2019-06-15 - [An introduction to ASGI, Asynchronous Server Gateway Interface](https://www.youtube.com/watch?v=t3gCK9QqXWU), P G Jones, PyLondinium 2019.
 - 2019-04-12 - [Sketching out A Django redesign](https://www.youtube.com/watch?v=u8GSFEg5lnU), Tom Christie, DjangoCon Europe.
+- 2018-07-27 - [Quart; an ASGI alternative to Flask](https://www.youtube.com/watch?v=t8-Y7Kivuu0), P G Jones, EuroPython 2018
```

Comment:
Can we stick to the original name of the talk? Also formatting nit: final full stop is missing. :)

Reply:
Seems like EuroPython labeled it incorrectly. [Original name of the talk](https://i.imgur.com/47OcSKb.png) :-)

Full stop added.